src/main/java/aeon/controller/Command.java

```diff
@@ -0,0 +1,297 @@
+package aeon.controller;
+import java.io.IOException;
+import java.sql.Array;
+import java.util.Scanner;
+import aeon.task.Task;
+import aeon.task.Event;
+import aeon.task.Todo;
+import aeon.task.Deadline;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.util.ArrayList;
+
+
+public class Command {
+
+    public static final String TASK_NOT_FOUND = "Task not found! Perhaps try listing out the available tasks first...";
+    public static final String INVALID_INTEGER_MSG = "Please type in a valid integer!";
+    public static final String TODO_DESC_ERROR = "Description of TODO cannot be empty!!!";
+    public static final String DEADLINE_FORMAT_ERR = "Please try again in this format: deadline /by <date>";
+    public static final String EVENT_FORMAT_ERR = "Please try again in this format: event /at <date>";
+    public static final String INVALID_COMMAND = "Not sure what you were trying to do...";
+    public static final String TASK_ADDED = "Task added!";
+    public static final String CONGRATULATIONS_MSG = "Congrats on completing this task!:";
+    public static final String MARK_UNDONE = "Alright, marked as undone!:";
+    public static final String NO_TASKS = "No tasks!";
+    public static final String FILE_PATH = "./data/tasklist.txt";
+    public static final String DIR_PATH = "./data/";
+
+    public static final String CREATE_FILE_FAILED = "Failed to create file to store task!";
+
+    private static ArrayList<String> rawDescList = new ArrayList<>();
```

Comment:
According to the Java coding standards, 

> Plural form should be used on names representing a collection of objects.

Consider renaming the variable (e.g. `rawDescriptions`)